movie_finder_django/-r

```diff
@@ -8,7 +8,6 @@ distro==1.9.0
 Django==5.0.7
```

Comment:
fix files please. You have renamed requirements.txt to -r and created new file instead of updating original one